refinery/templates/base.html

```diff
@@ -322,6 +322,9 @@
   <script src="{% static "vendor/jquery-file-upload/js/jquery.fileupload-angular.js" %}"></script>
   <script src="{% static "js/data-set-import.js" %}"></script>
 
+  <!-- Fetch external SVGs referenced in <use> elements when the browser fails to do so -->
+  <script defer src="{% static "vendor/svgxuse/svgxuse.js" %}"></script>
```

Comment:
It works (locally at least) the `defer` was in their usage docs so I utilized as well.